.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -199,4 +215,76 @@ phases:
         displayName: Setup Build Variables
       - template: ./templates/.builder-release-template.yaml
         parameters:
-          artifactName: 1804-gen2-gpu-containerd
\ No newline at end of file
+          artifactName: 1804-gen2-gpu-containerd
+  - phase: build_vhd_1804_fips_containerd
+    condition: eq('${{ parameters.build1804fipscontainerd }}', true)
+    queue:
+      name: Hosted Ubuntu 1604
+      timeoutInMinutes: 180
+    steps:
+      - bash: |
+          echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
+          echo '##vso[task.setvariable variable=UBUNTU_SKU]18.04'
```

Comment:
There will be conflict with Mariner PR https://github.com/Azure/AgentBaker/pull/656
please watch/coordinate

Reply:
yes, a lot of conflicts, need to test again